doc/classes/TabBar.xml

```diff
@@ -244,6 +259,9 @@
 		<member name="tab_count" type="int" setter="set_tab_count" getter="get_tab_count" default="0">
 			The number of tabs currently in the bar.
 		</member>
+		<member name="tab_input_rollover" type="bool" setter="set_tab_input_rollover" getter="get_tab_input_rollover" default="false">
+			If [code]true[/code], pressing [code]ui_right[/code] on the last tab will select the first tab and pressing [code]ui_left[/code] on the first tab will select the last one.
+		</member>
```

Comment:
So when I suggested `tab_input_rollover` I only meant it for the `TabContainer`, not for the underlying `TabBar`. Here it should probably be just `input_rollover`. Just like `focus_mode` is just `focus_mode here and `tab_focus_mode` in the proxy property of the container.